<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="X-UA-Compatible" content="ie=edge">
<title>Markmap</title>
<style>
* {
  margin: 0;
  padding: 0;
}
#mindmap {
  display: block;
  width: 100vw;
  height: 100vh;
}
</style>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.17.0/dist/style.css">
</head>
<body>
<svg id="mindmap"></svg>
<script src="https://cdn.jsdelivr.net/npm/d3@7.8.5/dist/d3.min.js"></script><script src="https://cdn.jsdelivr.net/npm/markmap-view@0.17.0/dist/browser/index.js"></script><script src="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.17.0/dist/index.js"></script><script>((r) => {
            setTimeout(r);
          })(() => {
  const { markmap, mm } = window;
  const toolbar = new markmap.Toolbar();
  toolbar.attach(mm);
  const el = toolbar.render();
  el.setAttribute("style", "position:absolute;bottom:20px;right:20px");
  document.body.append(el);
})</script><script>((getMarkmap, getOptions, root2, jsonOptions) => {
              const markmap = getMarkmap();
              window.mm = markmap.Markmap.create(
                "svg#mindmap",
                (getOptions || markmap.deriveOptions)(jsonOptions),
                root2
              );
            })(() => window.markmap,null,{"content":"Computational Biology Tools","children":[{"content":"Programming Languages","children":[{"content":"Python","children":[{"content":"Libraries","children":[{"content":"<code>pandas</code> <a href=\"https://pandas.pydata.org/docs/\">docs</a>","children":[],"payload":{"lines":"4,5"}},{"content":"<code>numpy</code> <a href=\"https://numpy.org/doc/stable/\">docs</a>","children":[],"payload":{"lines":"5,6"}},{"content":"<code>scipy</code> <a href=\"https://docs.scipy.org/doc/scipy/reference/\">docs</a>","children":[],"payload":{"lines":"6,7"}},{"content":"<code>matplotlib</code> <a href=\"https://matplotlib.org/stable/contents.html\">docs</a>","children":[],"payload":{"lines":"7,8"}},{"content":"<code>seaborn</code> <a href=\"https://seaborn.pydata.org/\">docs</a>","children":[],"payload":{"lines":"8,9"}},{"content":"<code>scikit-learn</code> <a href=\"https://scikit-learn.org/stable/documentation.html\">docs</a>","children":[],"payload":{"lines":"9,10"}},{"content":"<code>biopython</code> <a href=\"https://biopython.org/docs/1.75/api/\">docs</a>","children":[],"payload":{"lines":"10,11"}}],"payload":{"lines":"3,11"}}],"payload":{"lines":"2,3"}},{"content":"R","children":[{"content":"Libraries","children":[{"content":"<code>ggplot2</code> <a href=\"https://ggplot2.tidyverse.org/\">docs</a>","children":[],"payload":{"lines":"13,14"}},{"content":"<code>tidyverse</code> <a href=\"https://www.tidyverse.org/\">docs</a>","children":[{"content":"<code>dplyr</code> <a href=\"https://dplyr.tidyverse.org/\">docs</a>","children":[],"payload":{"lines":"15,16"}}],"payload":{"lines":"14,16"}},{"content":"<code>caret</code> <a href=\"https://topepo.github.io/caret/index.html\">docs</a>","children":[],"payload":{"lines":"16,17"}},{"content":"<code>bioconductor</code> <a href=\"https://www.bioconductor.org/\">docs</a>","children":[],"payload":{"lines":"17,18"}}],"payload":{"lines":"12,18"}}],"payload":{"lines":"11,12"}},{"content":"Perl","children":[{"content":"Libraries","children":[{"content":"<code>BioPerl</code> <a href=\"https://bioperl.org/\">docs</a>","children":[],"payload":{"lines":"20,21"}}],"payload":{"lines":"19,21"}}],"payload":{"lines":"18,19"}}],"payload":{"lines":"1,2"}},{"content":"Pipelines","children":[{"content":"Slurm","children":[{"content":"<code>srun</code> <a href=\"https://slurm.schedmd.com/srun.html\">docs</a>","children":[],"payload":{"lines":"23,24"}},{"content":"<code>sbatch</code> <a href=\"https://slurm.schedmd.com/sbatch.html\">docs</a>","children":[],"payload":{"lines":"24,25"}},{"content":"You can get pretty far just using bash with these two commands, but at some point managing the input/output of a whole graph of tasks gets cumbersome and the next two options are better.","children":[],"payload":{"lines":"25,26"}}],"payload":{"lines":"22,23"}},{"content":"Nextflow","children":[{"content":"<a href=\"https://www.nextflow.io/\">Nextflow</a> is a workflow management system that enables the writing of scripts in a domain-specific language to define the workflow. Notably it works with containers both Docker and Singularity. This can also use slurm.","children":[],"payload":{"lines":"27,28"}}],"payload":{"lines":"26,27"}},{"content":"Snakemake","children":[{"content":"<a href=\"https://snakemake.readthedocs.io/en/stable/\">Snakemake</a> is a workflow management system that allows the creation of reproducible and scalable data analyses. This can also use slurm.","children":[],"payload":{"lines":"29,30"}}],"payload":{"lines":"28,29"}}],"payload":{"lines":"21,22"}},{"content":"RNA-Seq Analysis","children":[{"content":"Alignment","children":[{"content":"<code>STAR</code> <a href=\"https://code.google.com/archive/p/rna-star/\">docs</a>","children":[],"payload":{"lines":"32,33"}},{"content":"<code>tophat</code> <a href=\"https://ccb.jhu.edu/software/tophat/index.shtml\">docs</a>","children":[],"payload":{"lines":"33,34"}},{"content":"<code>bowtie 2</code> <a href=\"https://github.com/BenLangmead/bowtie2\">docs</a>","children":[],"payload":{"lines":"34,35"}}],"payload":{"lines":"31,32"}}],"payload":{"lines":"30,31"}},{"content":"Biological File Formats","children":[{"content":"SAM/BAM Tools","children":[{"content":"<code>samtools</code> <a href=\"http://www.htslib.org/doc/samtools.html\">docs</a>","children":[],"payload":{"lines":"37,38"}}],"payload":{"lines":"36,37"}},{"content":"VCF Tools","children":[{"content":"<code>vcftools</code> <a href=\"https://vcftools.github.io/index.html\">docs</a>","children":[],"payload":{"lines":"39,40"}}],"payload":{"lines":"38,39"}},{"content":"BED Tools","children":[{"content":"<code>bedtools</code> <a href=\"https://bedtools.readthedocs.io/en/latest/\">docs</a>","children":[],"payload":{"lines":"41,42"}}],"payload":{"lines":"40,41"}},{"content":"FASTA/FASTQ Tools","children":[],"payload":{"lines":"42,43"}},{"content":"GFF/GTF Tools","children":[],"payload":{"lines":"43,44"}}],"payload":{"lines":"35,36"}},{"content":"Other resources","children":[{"content":"<a href=\"https://github.com/danielecook/Awesome-Bioinformatics\">Awesome Bioinformatics</a>","children":[],"payload":{"lines":"45,46"}},{"content":"<a href=\"https://bio.tools/\">bio.tools</a>","children":[],"payload":{"lines":"46,47"}}],"payload":{"lines":"44,45"}}],"payload":{"lines":"0,1"}},{"colorFreezeLevel":2,"maxWidth":300,"embedAssets":true})</script>
</body>
</html>
